truenas/webui
# NAS-115291: Adding force flag to unlock operation

Testing: 
1. Create a dataset.
2. Edit dataset encryption options and change to password.
3. Lock dataset.
4. Unlock and observe Force flag.
